pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -11,21 +11,31 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/hyperledger/aries-framework-go/pkg/storage"
 )
 
 const (
-	keyPattern   = "%s_%s"
-	invKeyPrefix = "inv_"
+	keyPattern        = "%s_%s"
+	invKeyPrefix      = "inv"
+	connIDKeyPrefix   = "conn"
+	protocolKeyPrefix = "pro"
 )
 
 // ConnectionRecord contain info about did exchange connection
 type ConnectionRecord struct {
 	// State of the connection invitation
-	State string
-
-	ConnectionID string
+	State           string
+	ConnectionID    string
+	ThreadID        string
+	RequestID       string
+	MyDID           string
+	TheirDID        string
+	TheirLabel      string
+	ServiceEndPoint string
+	RecipientKeys   []string
+	InvitationID    string
 }
```

Comment:
we need create & last update time, add a TODO if it too much for this PR.

Reply:
Timing is different decorater which need to embded and that should be part of that pr